cns/restserver/ipam.go

```diff
@@ -25,7 +25,8 @@
 	)
 
 	err = service.Listener.Decode(w, r, &ipconfigRequest)
-	logger.Request(service.Name, &ipconfigRequest, err)
+	operationName := "requestIPConfigHandler"
+	logger.Request(service.Name+operationName, ipconfigRequest.String(), err)
```

Comment:
You dont have to add String()

Reply:
I need add String, otherwise, the OrchestratorContext part is binary and not readable. String() function translate the binary to string for OrchestratorContext part.
![image](https://user-images.githubusercontent.com/16043262/111389122-a43dc180-866d-11eb-8402-6934398cc43a.png)

Reply:
Tested, no need to call .String specifically. Let me remove it.